cypress/integration/group-membership.spec.js

```diff
@@ -0,0 +1,101 @@
+describe( 'Course Settings', () => {
+	context( 'Group Admin', () => {
+		before( () => {
+			cy.logout();
+			cy.login('mamaduka');
```

Comment:
I assume this should not be hardcoded?

Reply:
Sorry about this 😅already fixed with 099595b.